modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Comment.es.js

```diff
@@ -63,6 +72,9 @@ export default withRouter(
 					<div className="c-mb-0">
 						<ArticleBodyRenderer
 							{...comment}
+							companyMxNameLabel={
+								comment.companyMxName && companyMxNameLabel
+							}
```

Comment:
This validation is strange, are you passing a string or a boolean? 


If you want to use the value from `comment.companyMxName` or `companyMxNameLabel`

You could use

```
comment.companyMxName ?? companyMxNameLabel
```

```suggestion
							companyMxName={
								comment.companyMxName && companyMxNameLabel
							}
```

Reply:
The change to "??" did not work.